firestore.rules

```diff
@@ -9,5 +9,8 @@ service cloud.firestore {
     match /{document=**} {
       allow read, write: if false;
     }
+    match /sources/{source}/resources/{document=**} {
+      allow read: if request.auth.uid != null && request.auth.token.firebase.sign_in_provider != 'anonymous';
```

Comment:
@pjanik are we planning to narrow these back down use the class hash or class id?

Reply:
I don't think we want to do that for "resources" => activity/sequence structure.
Activity structure doesn't contain any private data. We could be concerned about students getting access to it and checking correct answers, but using class_hash won't help with that anyway. 

The plan was to use clash_hash or class_url for rules related to the student answers.